drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -44,12 +44,9 @@ def client_search(input:, **args)
     end
 
     def client_omni_search(text_search:)
-      client_scope = Hmis::Hud::Client.searchable_to(current_user).
+      Hmis::Hud::Client.searchable_to(current_user).
         matching_search_term(text_search).
-        includes(:enrollments).
-        order(qualified_column(e_t[:date_updated]))
-
-      resolve_clients(client_scope, no_sort: true)
+        sort_by_option(:recently_added)
```

Comment:
unrelatedly, removing this really slow sort. should be addressed by https://www.pivotaltracker.com/story/show/185680669 hopefully.